docs/adr/0002-BuildWithCliTool.md

```diff
@@ -0,0 +1,23 @@
+# Build with CLI tool

```

Comment:
I would not include ADRs in the GitRepo

Reply:
Dropped in e36ea6731206b710e736dd335e93233de553801f